fullcalendar-app/src/components/FullCalendar.vue

```diff
@@ -172,41 +175,73 @@ export default {
         this.calendarOptions.slotDuration = window.drupalSettings.fullCalendar.slotDuration;
         this.calendarOptions.snapDuration = window.drupalSettings.fullCalendar.snapDuration;
         this.calendarOptions.slotLabelInterval = window.drupalSettings.fullCalendar.slotLabelInterval;
+
+        // Also update minTime and maxTime
+        this.calendarOptions.slotMinTime = window.drupalSettings.fullCalendar.minTime || '05:00:00';
```

Comment:
Let's have default the same as in config

Reply:
Oh right, good point

Reply:
Fixed.